tools/clang/lib/SPIRV/SpirvBuilder.cpp

```diff
@@ -254,9 +269,32 @@ SpirvLoad *SpirvBuilder::createLoad(const SpirvType *resultType,
 SpirvStore *SpirvBuilder::createStore(SpirvInstruction *address,
                                       SpirvInstruction *value,
                                       SourceLocation loc, SourceRange range) {
-  assert(insertPoint && "null insert point");
```

Comment:
Any particular reason for removing this?

Reply:
No, mistake from the change, restoring this!